### ISSUE TITLE
save_image import error

### ISSUE THREAD

**Issue by jemshid** [Eric-Mendes/unexpected-isaves#93]
**Describe the bug**
Error when importing save_image

**To Reproduce**
```
$ python3
Python 3.8.10 (default, Mar 13 2023, 10:26:41) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import save_image as si
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/j/Dropbox/cube/unexpected-isaves/src/unexpected_isaves/save_image.py", line 106, in <module>
    res: list[str],
TypeError: 'type' object is not subscriptable
```

**Expected behavior**
```
>>> import save_image as si
>>>
``` 

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.4 LTS

**Comment by jemshid** [comment]
Fix for the issue is here.
[](https://github.com/jemshid/unexpected-isaves/blob/main/src/unexpected_isaves/save_image.py)

**Comment by Eric-Mendes** [comment]
Thanks for pointing it out! Do you want to open a merge request or do you prefer that I fix it?

**Comment by jemshid** [comment]
@Eric-Mendes Fixed the issue and created a pull request.

**Comment by Eric-Mendes** [comment]
Closed with #94 & #95.
